src/comps/main/main.jsx

```diff
@@ -7,6 +7,24 @@ import { AddComments } from "./addComments/addComments";
 import { allVideos } from "../../data/UserFunctions";
 import main from "../main/main.css";
 const Main = () => {
+  const styles = {
+    img: {
+      width: "200px",
+      marginTop: "10px",
+      cursor: "pointer"
+    },
+    activeVid: {
+      height: "415px",
+      width: "700px"
+    },
+    div1: {
+      width: "700px",
+      height: "100px",
+      marginLeft: "60px",
+      color: "black"
+    }
+  };
+
   const [activeVideo, setActiveVideo] = useState({
     id: 0,
     key: 0,
```

Comment:
Just put the content of getAllVideos inside the useEffect, you shouldn't declare functions inside the render.
What's allVideos? sounds like an array but is being called like a function.